src/main/kotlin/io/github/seggan/jyxal/runtime/RuntimeMethods.kt

```diff
@@ -273,6 +281,35 @@ fun doubleRepeat(obj: Any): Any {
     }
 }
 
+fun ezr(obj: Any): Any {
+    return when (obj) {
+        is JyxalList -> {
```

Comment:
yeah, once again, if you specify that monads are vectorised in `Element`, then the compiler will auto insert the vectorisation. Might want to remove this `is JyxalList`